UOP1_Project/Assets/Scripts/CameraManager.cs

```diff
@@ -9,6 +9,15 @@ public class CameraManager : MonoBehaviour
 	public InputReader inputReader;
     public CinemachineFreeLook freeLookVCam;
 
+ 	[Tooltip("General multiplier for camera sensitivity/speed")]
+	public float cameraSensitivity = 7.0f;
+
+ 	private void Awake()
+    {
+		// Don't allow the camera to be slower than default and then also 20+ gets pretty crazy.
+        cameraSensitivity = Mathf.Clamp(cameraSensitivity, 1.0f, 20.0f);
```

Comment:
I dont think we need this sanity check. One should not commit/approve a change in the `cameraSensitivity` if it breaks the camera. 

You can still "tell" people that you want them to use a value between 1 and 20 by using a range attribute over the `cameraSensitivity` field. It will make people hesitant to use values outside the range and keep the code clean.

Reply:
Cool. Good suggestion. I'd forgotten about the range option in Unity. I'll do this.